setup.py

```diff
@@ -121,6 +121,7 @@
     'jsonresolver>=0.2.1',
     'marshmallow>=2.5.0',
     'Pillow>=3.2.0',
+    'pexpect>=4.2.1',
```

Comment:
Maybe we should also pin `invenio-webhooks` to the current version to avoid problems when these changes start getting integrated mainstream